cmd/destroy.go

```diff
@@ -133,7 +133,7 @@ func newDestroyCmd() *cobra.Command {
 		"Display operation as a rich diff showing the overall change")
 	cmd.PersistentFlags().IntVarP(
 		&parallel, "parallel", "p", defaultParallel,
-		"Allow P resource operations to run in parallel at once (1 for no parallelism, 0 for unbounded parallelism)")
+		"Allow P resource operations to run in parallel at once (1 for no parallelism). Defaults to unbounded.")
```

Comment:
Can we keep the command-line option's behavior the same and just tweak this internally?